README.md

```diff
@@ -69,12 +69,53 @@ Standard Usage:
 - changes to **persistStore**:
   - 1. remove config argument (or replace with an empty object)
   - 2. remove all arguments from the callback. If you need state you can call `store.getState()`
-  - 3. All constants (ex: `{REHYDRATE, PURGE}`) has moved to `redux-persist/lib/constants` instead of `redux-persist/constants`
-- replace combineReducers with **persistCombineReducers**
+  - 3. All constants (ex: `REHYDRATE`, `PURGE`) has moved to `redux-persist/lib/constants` instead of `redux-persist/constants`
+- replace `combineReducers` with **persistCombineReducers**
   - e.g. `let reducer = persistCombineReducers(config, reducers)`
 - changes to **config**:
   - `key` is now required. Can be set to anything, e.g. 'primary'
   - `storage` is now required. For default storage: `import storage from 'redux-persist/lib/storage'`
+  
+```diff
+-import { REHYDRATE, PURGE } from 'redux-persist/constants'
++import { REHYDRATE, PURGE } from 'redux-persist/lib/constants'
```

Comment:
I am going to move constants into the root in the next release

Reply:
Depends on how fast you'll make a new major release I guess. We can always change this again when you make a new release 😄

Reply:
ok its up v5.3.3, so this line can be
`import { REHYDRATE, PURGE } from 'redux-persist'`

The old format still works but this is I think preferred now.